web/styles/rendered_markdown.css

```diff
@@ -639,8 +643,12 @@
         & video {
             display: block;
             object-fit: contain;
-            height: 100%;
-            width: 100%;
+            /* Since we cannot get height and width
+               values for media thumbnails, we now
+               carry forward the legacy thumbnail
+               dimensions here. */
```

Comment:
I think this comment would do better to not say "legacy thumbnail dimensions" but instead describe itself as a reasonable landscape card size appropriate for video thumbnails.

Reply:
Agreed. Fixed.